tests/drivers/build_all/video/testcase.yaml

```diff
@@ -13,4 +13,6 @@ tests:
       - gpio
       - i2c
   drivers.video.mcux_csi.build:
```

Comment:
We shouldn't be adding the RT1170 to the CSI build test until after we add support to the driver, right? So this change shouldn't be the commit adding the ov5640 to the build all test

Reply:
done